test/Dialect/SPIRV/Serialization/bin_ops.mlir

```diff
@@ -0,0 +1,52 @@
+// RUN: mlir-translate -serialize-spirv %s | mlir-translate -deserialize-spirv | FileCheck %s
+
+func @spirv_bin_ops() -> () {
+  spv.module "Logical" "VulkanKHR" {
+   func @fmul(%arg0 : f32, %arg1 : f32) {
+      // CHECK: [[VALUE:%.*]] = spv.FMul [[ARG1:%.*]], [[ARG2:%.*]] : f32
```

Comment:
Nit: This doesnt need to be [[..:%.*]]. Just use {{%.*}} since these values arent matched anywhere else

Reply:
Thanks, fixed!